From 41db6d93ab4d91f74761f8a34800b9026b0482c7 Mon Sep 17 00:00:00 2001
From: Sebastian Pipping <sebastian@pipping.org>
Date: Sat, 6 Jun 2015 20:25:39 +0200
Subject: [PATCH] Fix underquoting in configure.ac

.. resulting in

  checking for glXSwapBuffers in -lGL... ./configure: line 3095: ac_fn_c_try_link: command not found
  no

at runtime.
---
 configure.ac | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/configure.ac b/configure.ac
index 7bb7e72..cd7fa54 100644
--- a/configure.ac
+++ b/configure.ac
@@ -54,18 +54,18 @@ CYGWIN*|cygwin*|MINGW*|mingw*)
     OPT_OBJS="main.o canvas_base.o"
     PROGRAM="fireflies"
 
-    AC_CHECK_LIB(GL, glXSwapBuffers,\
-	AC_DEFINE(HAVE_GLX)
-	OPT_OBJS="$OPT_OBJS canvas_glx.o",\
-	AC_CHECK_LIB(MesaGL, glXSwapBuffers,\
-	    AC_DEFINE(HAVE_GLX)
-	    OPT_OBJS="$OPT_OBJS canvas_glx.o"))
-
-    AC_CHECK_LIB(GL, glBegin, \
-	GL_LIBS="-lGL -lGLU", \
-	AC_CHECK_LIB(MesaGL, glBegin, \
+    AC_CHECK_LIB([GL], [glXSwapBuffers],\
+	[AC_DEFINE(HAVE_GLX)
+	OPT_OBJS="$OPT_OBJS canvas_glx.o"],\
+	[AC_CHECK_LIB([MesaGL], [glXSwapBuffers],\
+	    [AC_DEFINE(HAVE_GLX)
+	    OPT_OBJS="$OPT_OBJS canvas_glx.o"])])
+
+    AC_CHECK_LIB([GL], [glBegin], \
+	[GL_LIBS="-lGL -lGLU"], \
+	[AC_CHECK_LIB([MesaGL], [glBegin], \
 	    [GL_LIBS="-lMesaGL -lMesaGLU"], \
-	    AC_MSG_ERROR([cannot find GL libraries])))
+	    [AC_MSG_ERROR([cannot find GL libraries])])])
 
     ;;
 esac
-- 
2.4.0

